oauth2/provider/login.php

```diff
@@ -66,11 +69,32 @@
                         </div>
                         <?php } ?>
 
-                        <?php if (empty($authorize) && $mfaRequired && $mfaType === 'TOTP') { ?>
+                        <?php if (empty($authorize) && $mfaRequired ) { ?>
```

Comment:
again I think PHP wants explicite conditionals from what I've seen Brady fix many in core. e.q !empty($mfaRequired) I could be wrong here as I haven't researched.